lib/Conversion/TorchToLinalg/Utils.cpp

```diff
@@ -67,6 +67,21 @@ Value torch_to_linalg::getPaddedTensor(Operation *op, OpBuilder &b,
   return getPaddedTensor(op, b, input, paddingInts, paddingInts, c0);
 }
 
+Value torch_to_linalg::getPaddedTensor(
+    Operation *op, OpBuilder &b, Value &input,
+    SmallVectorImpl<int64_t> &lowPaddingInts,
+    SmallVectorImpl<int64_t> &highPaddingInts, Value pad, Type outType) {
+  Location loc = op->getLoc();
+  SmallVector<OpFoldResult> lowPaddings =
+      getIndexIntsAsOpFoldResult(b, lowPaddingInts);
+  SmallVector<OpFoldResult> highPaddings =
+      getIndexIntsAsOpFoldResult(b, highPaddingInts);
+  Value paddedInput = tensor::createPadScalarOp(
+      outType, input, pad, /*low=*/lowPaddings, /*high=*/highPaddings,
+      /*packing=*/false, loc, b);
+  return paddedInput;
+}
```

Comment:
This is almost the exact same code as `getPaddedTensor` above. Can you write one in terms of the other?